curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c13b3c7e00e01c0e73d52.md

```diff
@@ -0,0 +1,241 @@
+---
+id: 643c13b3c7e00e01c0e73d52
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+Without passing a URL argument to the `fetch` function, the `fetch` method will not know where to fetch the data from.
+
+For your `fetch()` call, provide the `FORUM_LATEST` constant as an argument.
+
+All of the URLs you will need have been declared for you using the screaming snake case naming convention. This is a popular convention for naming constants to communicate to other developers that the variable should not be changed.
```

Comment:
we are going to rename all instances of snake case here.
Let's remove this paragraph

```suggestion
```

Reply:
removed